tests/__init__.py

```diff
@@ -1,4 +1,4 @@
-DUBLIN_CORE_XML = '''<?xml version="1.0" encoding="UTF-8"?>
+DUBLIN_CORE_XML = '''<?xml version="1.0"?>
```

Comment:
Don't we still need to give the encoding?

Reply:
I think I should have used encode() function on `DUBLIN_CORE_XML` elements instead of removing the encoding in the xml file.